src/main/kotlin/no/nav/familie/baks/mottak/task/JournalhendelseBarnetrygdRutingTask.kt

```diff
@@ -66,7 +66,17 @@ class JournalhendelseBarnetrygdRutingTask(
             return
         }
 
-        val personIdent = journalpostBrukerService.tilPersonIdent(journalpost.bruker, tema)
+        val personIdent =
+            try {
+                journalpostBrukerService.tilPersonIdent(journalpost.bruker, tema)
+            } catch (error: NoSuchElementException) {
```

Comment:
Antar `NoSuchElementException` kommer fra `.last` metoden, men her hadde det nok vært mer fornuftig med en mer eksplisitt  exception. Kan man, i metoden som kaller `.last`, legge til en sjekk for tom liste, og hvis listen er tom kaste en egendefinert PDL exception?

Reply:
Enig i denne. Og husk at samme feil kan skje i JournalhendelseKontantstøtteRutingTask.


En liten kommentar på siden.
At man faktisk har en journalpost med en aktør hvor man får tom respons fra pdl med identer er veldig veldig spesielt og nok ikke noe som kommer til å skje ofte. Første gang jeg har sett. Så i dette tilfellet så kunne man nok bare ha avviskhåndtert tasken. Da ville sikkerhetsnettet til Joark/dokument ha plukket opp journalposten og lagd journalføringsoppgave på den.

Reply:
Det er først etter endringene vi har gjort med [automatisk journalføring](https://github.com/navikt/familie-baks-mottak/commit/17ed41761eab5c0e08e5dd58afb98abaa9d1c160#diff-93fdc8dffc573e0ddbe5b815e13f357d96ad2162d97aa13e498661e74217d70cR73) som gjør at dette feiler, før det hadde vi ikke oppslag i PDL akkurat her. Jeg er enig at vi burde ha avvikshåndtert den hvis dette er første gang denne har feilet på kode som har kjørt veldig lenge, men det er ikke tilfelle her. Det gir dermed mening at du aldri har sett feilen før @stigebil. Jeg kan fikse en egen type feilmelding 😃